CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Add support for the DeliverMax field in Payment transactions
+- Use rippled API v2 as default in requests
```

Comment:
I think this should be at the top (or highlighted some how) since this is a major change. Other than that, LGTM.

Reply:
fixed in 4179077